client/js/lounge.js

```diff
@@ -1039,9 +1042,7 @@ $(function() {
 			socket.emit("names", {target: self.data("id")});
 		}
 
-		if (screen.width > 768 && chan.hasClass("chan")) {
-			input.focus();
-		}
+		focus();
```

Comment:
> That means it worked as intended.

My bad then, for some reason I thought the intent was different.

> You want to keep the keyboard open after sending message? Or at least focus on the input without doing touch check?

Indeed, it seems counter-intuitive at the moment. I realize that *every time* I send a message, I expect the keyboard to stick around. This is probably because all other messaging apps I use on my phone (Slack, Telegram, SMS) do that.
I think we should do that as well after sending a message, setting/canceling nick, and cycling through nicks. But when changing channels, we should definitely not open the keyboard 😅